src/main/java/net/mcreator/ui/minecraft/boundingboxes/JBoundingBoxList.java

```diff
@@ -31,13 +32,12 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class JBoundingBoxList extends JPanel {
+public class JBoundingBoxList extends JEntriesList {
 	private final List<JBoundingBoxEntry> boundingBoxList = new ArrayList<>();
 	private final JPanel entries = new JPanel(new GridLayout(0, 1, 5, 5));
-	private final JButton add = new JButton(UIRES.get("16px.add.gif"));
 
 	public JBoundingBoxList(MCreator mcreator) {
-		super(new BorderLayout());
+		super(mcreator, new BorderLayout(), IHelpContext.NONE);

```

Comment:
Why are you using NONE context here, and not GUI like in other entry lists?

Reply:
Is help context used by bounding box entries?

Reply:
Even if it is not for now, you should put it here for the sake of consistency..